app/styles/_projects.less

```diff
@@ -0,0 +1,66 @@
+//
+// Projects page
+// --------------------------------------------------
+
+.project-bar {
+  .project-sort {
+    .form-group {
+      margin: 0 0 0 0;
+    }
+  }
+
+  .projectSearch {
+    flex: 1;
+
+    @media (min-width: @screen-sm-min) {
+      flex: 0;
+      min-width: 200px;
+    }
+  }
+
+  .vertical-divider {
+    // Controls are stacked at mobile, so hide the divider.
+    display: none;
+  }
+  .vertical-divider {
+    // @gray-light is too dark, and @gray-lighter is too light.
+    // Choose another gray from the Patternfly palette.
+    //   https://www.patternfly.org/styles/color-palette/
+    border-left: 1px solid #bbbbbb;
+    display: inline-block;
+    height: 25px;
+    margin: 0 7px;
+    vertical-align: middle;
+    width: 1px;
+  }
+}
+
+.project-names {
+  min-width: 35%;
+}
+
+.tile-project {
```

Comment:
want to stick with the convention of `project-name`, `project-tile`, `project-*`? rather than reversing to `tile-project` here.